homeassistant/components/homeassistant/strings.json

```diff
@@ -27,6 +27,10 @@
     "no_platform_setup": {
       "title": "Unused YAML configuration for the {platform} integration",
       "description": "It's not possible to configure {platform} {domain} by adding `{platform_key}` to the {domain} configuration. Please check the documentation for more information on how to set up this integration.\n\nTo resolve this:\n1. Remove `{platform_key}` occurences from the `{domain}:` configuration in your YAML configuration file.\n2. Restart Home Assistant.\n\nExample that should be removed:\n{yaml_example}\n"
+    },
+    "storage_corruption": {
+      "title": "Storage corruption detected for `{key}`",
+      "description": "The storage of `{key}` is corrupted and has been archived as `{corrupt_path}`. Please check the log for more details and consider restoring from backup."
```

Comment:
To help translators, it would be clearer if instead of `key` and `corrupt_path` the placeholders have more similar names.

Reply:
changed to file_key and added original_path